.github/workflows/test-e2e-runtime-watcher.yml

```diff
@@ -68,7 +72,6 @@ jobs:
       - name: Additional test cluster configuration
         run: |
           k3d cluster edit skr --port-add 9090:9090@loadbalancer
-          k3d cluster edit skr --port-add 2112:2112@loadbalancer
```

Comment:
Necessary because in the meantime this has been introduced to the base script creating the clusters in https://github.com/kyma-project/lifecycle-manager/pull/2226/files#diff-46bbbfcc5ff33de8ce4d58511664822348de25d52909bf999508c74a16fef47f